webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -188,6 +188,14 @@ <h2>{{ report.product }} {{ report.version }} Crash Report [@ {{ report.signatur
                     {% endif %}
                   </td>
                 </tr>
+                {% if report.product == "Fenix" %}
+                  <tr title="{{ fields_desc['processed_crash.application_build_id'] }}">
+                    <th scope="row">Application Build ID</th>
+                    <td>
+                      {{ report.application_build_id }}
+                    </td>
+                  </tr>
+                {% endif %}
```

Comment:
The ApplicationBuildID isn't structured the same as BuildID, plus Fenix isn't in Buildhub. So there's no point in showing a Buildhub link or extracting a date from the value.